.releaserc

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@efischer19 why specify this file? It seems like a duplicate of [the default plugins that `semantic-release`](https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/plugins.md#plugins) comes with?